src/core/connection.c

```diff
@@ -317,7 +317,9 @@ QuicConnFree(
     CXPLAT_TEL_ASSERT(Connection->SourceCids.Next == NULL);
     CXPLAT_TEL_ASSERT(CxPlatListIsEmpty(&Connection->Streams.ClosedStreams));
     QuicRangeUninitialize(&Connection->DecodedAckRanges);
+    QuicConfigurationAttachSilo(Connection->Configuration);
```

Comment:
If this guaranteed to be non-null?

Reply:
We also deref configuration in this file. So, it actually has to be non-null.